presto-main/src/main/java/io/prestosql/execution/CommentTask.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.execution;
+
+import com.google.common.util.concurrent.ListenableFuture;
+import io.prestosql.Session;
+import io.prestosql.metadata.Metadata;
+import io.prestosql.metadata.QualifiedObjectName;
+import io.prestosql.metadata.TableHandle;
+import io.prestosql.security.AccessControl;
+import io.prestosql.sql.analyzer.SemanticException;
+import io.prestosql.sql.tree.Comment;
+import io.prestosql.sql.tree.Expression;
+import io.prestosql.transaction.TransactionManager;
+
+import java.util.List;
+import java.util.Optional;
+
+import static com.google.common.util.concurrent.Futures.immediateFuture;
+import static io.prestosql.metadata.MetadataUtil.createQualifiedObjectName;
+import static io.prestosql.sql.analyzer.SemanticErrorCode.MISSING_TABLE;
+
+public class CommentTask
+        implements DataDefinitionTask<Comment>
+{
+    @Override
+    public String getName()
+    {
+        return "COMMENT";
+    }
+
+    @Override
+    public ListenableFuture<?> execute(Comment statement, TransactionManager transactionManager, Metadata metadata, AccessControl accessControl, QueryStateMachine stateMachine, List<Expression> parameters)
+    {
+        Session session = stateMachine.getSession();
+
+        if (statement.getType() == Comment.Type.TABLE) {
```

Comment:
We should thrown a PrestoException with UNSUPPORTED code, otherwise.

Reply:
Perhaps, `NOT_SUPPORTED` is right? I couldn't find `UNSUPPORTED` code.  Updated like this.
```java
        else {
            throw new PrestoException(NOT_SUPPORTED, "Unsupported comment type: " + statement.getType());
        }
```

Reply:
Yeah, that’s what I meant.